sdks/python/apache_beam/testing/benchmarks/inference/pytorch_image_classification_benchmarks.py

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+import logging
+
+from apache_beam.examples.inference import pytorch_image_classification
+from apache_beam.testing.load_tests.load_test import LoadTest
+from torchvision import models
+
+_PERF_TEST_MODELS = ['resnet50', 'resnet101', 'resnet152']
+_PRETRAINED_MODEL_MODULE = 'torchvision.models'
+
+
+class PytorchVisionBenchmarkTest(LoadTest):
+  def __init__(self):
+    # TODO (anandinguva): make get_namespace() method in RunInference static
+    self.metrics_namespace = 'RunInferencePytorch'
+    super().__init__(metrics_namespace=self.metrics_namespace)
+
+  def test(self):
+    pretrained_model_name = self.pipeline.get_option('pretrained_model_name')
+    if not pretrained_model_name:
+      raise RuntimeError(
+          'Please provide a pretrained torch model name.'
+          ' Model name must be from the module torchvision.models')
+    if pretrained_model_name == _PERF_TEST_MODELS[0]:
+      model_class = models.resnet50
+    elif pretrained_model_name == _PERF_TEST_MODELS[1]:
+      model_class = models.resnet101
+    elif pretrained_model_name == _PERF_TEST_MODELS[2]:
+      model_class = models.resnet152
```

Comment:
Maybe define `_PERF_TEST_MODELS` to be a dict mapping the string "resnet50" to the class? i.e. `_PERF_TEST_MODELS = {'resnet50': models.resnet50}` and do a retrieval from the lookup?

Reply:
SG. will change it